websites/N/Neon/presence.ts

```diff
@@ -0,0 +1,125 @@
+const presence = new Presence({
+  clientId: "837985880408457217"
+}),
+  timestamp = Math.floor(Date.now() / 1000);
+
+function findElement(tagName: string, className: string): Element {
+  return Array.from(
+    document.querySelectorAll(tagName)
+  ).find((x) => x.className.includes(className));
+}
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "key",
+    details: "Browsing...",
+    startTimestamp: timestamp
+  },
+  pathname = document.location.pathname;
+
+  switch(true) {
```

Comment:
Please don't use switch(true) statement.

Reply:
Why not?